themes/README.md

```diff
@@ -0,0 +1,61 @@
+# Hyprland Community Themes
+
+A collection of community-developed themes for use with Hyprland.
+
+# Applying a theme
+
+Themes in this repository are intended for drop-in use with Hyprland, and should be easy to apply while preserving your core configurations. To start using a theme, please follow the steps bellow:
+
+1. Clone this repository to your local machine: 
+```
+git clone --recursive https://www.github.com/hyprwm/contrib
+```
+
+2. Navigate to the theme folder with 
+```
+cd ./contrib/themes
+```
+
+3. Create the theme directory in your hyprland config folder. All themes in this repository will expect to be placed in this directory: 
+```
+mkdir ~/.config/hypr/themes
+```
+ 
+
+4. Find the folders for any themes you want to use and copy them to the themes folder in your hyprland config folder. For example, to use the Garden theme run:
+```
+cp -r ./garden ~/.config/hypr/themes/
+```
+
+5. Assuming that we are applying the Garden theme, add the following line to the end of your hyprland.conf: 
+```
+source=~/.config/hypr/themes/garden/theme.conf
+```
+
+Be advised that these themes **DO NOT** provide a complete hyprland configuration. You will still need to handle all basic user-specific configuration in your `hyprland.conf` such as monitor setup, input device configuration, keybinds, execs, etc. This is done so that your basic user preferences can be kept separate and portable accross various themes.
+
+# Contributing
+
+Contributions are welcome! Please refer to these guidelines for formatting your theme in order to provide a consistent user experience. 
+
+## Contribution Guidelines
+
+The goal of this repository is to provide a collection of themes/rices that may be easily dropped in to any user's hyprland configuration with minimal setup, and without overriding the user's core configurations. In order to facilitate drop-in use of these themes, please format your theme according to the following guidelines.
+
+1. Create a folder for your theme at `$HOME/.config/hypr/themes/<YOUR THEME>`. This is where you will place any and all files directly related to your theme, including:
+
+	- Theme-related Hyprland configs (See #2)
```

Comment:
What does `#2` refer to here?

Reply:
Referring to point number 2 immediately below in the readme. Probably not necessary.

Reply:
Something like `$EWW_CMD` would do the trick. @vaxerski can you implement it? You can export `HYPRLAND_CMD` which will hold info about how Hyprland was launched, e.g `hyprland -c ~/.local/config/hypr/hyprland.conf`.